src/make.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Function finds sum of integers
+ * @param {} - integers or function
+ * @return {}  resulted sum
+ */
+module.exports = function make() {
+  if (arguments.length === 0) return 0;
+  const array = [];
+  array.push(...arguments);
+  const f = (...arg) => {
+    array.push(...arg);
+    return typeof arg[arg.length - 1] !== 'function' ? f : array.slice(0, -1).reduce(arg[arg.length - 1], 0);
+  };
+  return f;
```

Comment:
@CuppaKe Check if the function work with an arguments witch is not a sum function.

Reply:
Looks like default value of accumuluator is unnecessary, right?

Reply:
after fixing can i merge or should wait for reviewing?